### ISSUE TITLE
[bug]: The expandable objects in the docs table don't have enough contrast

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5238]
### 🐞 Describe the bug

In the documentation table, sometimes there is an object expandable. The text on those does not have enough contrast. Such issue can be found on the [Internet header page](http://localhost:9000/?path=/docs/ebb11274-091b-4cb7-9a3f-3e0451c9a865--docs) for instance (link is local as it is a raw component).

### 📷 Screenshots

![Image](https://github.com/user-attachments/assets/a4d52c40-df1e-4e88-acc5-b87d1d440712)
